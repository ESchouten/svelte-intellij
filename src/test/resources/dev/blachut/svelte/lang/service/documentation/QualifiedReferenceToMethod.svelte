<script lang="ts">
  /**
   * Yes, umber is a word and it's a color!
   */
  class Umber {
    /**
     * Returns a string representing an umber in CSS hex notation.
     * @param _digits Number of digits after the hash. Must be in the range 6-6, inclusive.
     */
    toHex(_digits?: number): string {
      return "#635147";
    }
  }

  let local = new Umber();
</script>

{local.<caret>toHex()}