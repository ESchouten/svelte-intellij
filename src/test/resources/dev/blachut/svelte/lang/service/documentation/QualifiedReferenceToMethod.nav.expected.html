(method) Umber.toHex(_digits?: number): string<br/>[ide]<br/>(function) <span style="color:#7a7a43;">Umber.toHex</span><span style="">(_digits?:&#32;</span><span style="color:#000080;">number</span><span style="">):&#32;</span><span style="color:#000080;">string</span>